mw-webapp/src/component/table/LoadingContext.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {createContext, ReactNode, useContext, useState} from "react";
+
+type LoadingContextType = {
+  isLoading: boolean;
+  setLoading: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const LoadingContext = createContext<LoadingContextType | undefined>(undefined);
```

Comment:
Why you decide to use useContext?

Reply:
I didnt want to use any external state manager